Carthage Sample/iOS-sample-app/Masking/MaskingViewController.swift

```diff
@@ -0,0 +1,56 @@
+import UIKit
+import ContentsquareModule
+
+class MaskingViewController: UIViewController {
+
+    @IBOutlet private weak var publicLabel: UILabel!
+    @IBOutlet private weak var privateLabel: UILabel!
+
+    var useCase: MaskingUseCase = .defaultMasking
+    private let defaultMaskedTypes = [UITextField.self,
+                              UILabel.self,
+                              UITextView.self,
+                              UIButton.self,
+                              UIPickerView.self,
+                              UIImageView.self]
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        switch useCase {
+        case .defaultMasking:
+            resetDefaultMasking()
+        case .fullyUnmasked:
+            unmaskAll()
+        case .maskImages:
+            unmaskAll()
+            Contentsquare.mask(viewsOfType: UIImageView.self)
```

Comment:
I cannot see the image in the app on a iPhone 12 simulator

Reply:
You don't see it when running the app, or in the session replay?